src/mixins/I13nMixin.js

```diff
@@ -7,13 +7,14 @@
 
 var React = require('react');
 var ReactDOM = require('react-dom');
+
+var clickHandler = require('../utils/clickHandler');
 var debug = require('debug')('I13nMixin');
+var DebugDashboard = require('../utils/DebugDashboard');
+var I13nUtils = require('./I13nUtils');
+var listen = require('subscribe-ui-event/dist/lib/listen');
```

Comment:
agree, we can follow up this if `subscribe-ui-event` makes change according to this.